### ISSUE TITLE
[0.7.8] Circles converted in Inkscape to combined paths burn weirdly

### ISSUE THREAD

**Issue by Sophist-UK** [meerk40t/meerk40t#1362]
A row of stitch holes (which started as circles in Inkscape and were combined into a path - without being separately converted in Inkscape to paths first) is burned as a series of 90degree segments burned in wrong order / wrong direction.

![image](https://user-images.githubusercontent.com/3001893/193468030-dcb09339-80ec-42c9-8fdf-7c15126e3f8e.png)

Here is the SVG file [Circles screwed up](https://user-images.githubusercontent.com/3001893/193468127-11397373-5ade-4e96-a623-c9994bd84071.svg)

I do this sort of thing all the time - and indeed in the original file there are other very similar combined circles which don't do this.

I am unclear whether this is an Inkscape or MK issue but it also does it in 0.8.

**Comment by tatarize** [comment]
![weirdcircles2](https://user-images.githubusercontent.com/3302478/194596092-481b94b4-80a2-4644-9716-e59c4dec2e07.svg)

Minimized out it's that there's a matrix which multiplies by -1, -1 for width and scale. I've seen a similar bug before but I thought I had fixed it. But, apparently not given this example.

**Comment by tatarize** [comment]
meerk40t/svgelements#50

Somehow this is flipping the arc and sweeps flags, I'll add a test for it at the minimum.

**Comment by tatarize** [comment]
```python
    def test_issue_mk_1362(self):
        circle = Circle(cx="-1.5", cy="-1.5", r="0.5")
        circle *= "matrix(0,-1,-1,0,0,0)"
        path = Path(circle)
        path_d = path.d()
        circle_d = circle.d()
        self.assertEqual(path_d, circle_d)
```
This test ultimately fails.

Expected :M 1.5,1 A 0.5,0.5 -90 0,1 1,1.5 A 0.5,0.5 -90 0,1 1.5,2 A 0.5,0.5 -90 0,1 2,1.5 A 0.5,0.5 -90 0,1 1.5,1 Z
Actual   :M 1.5,1 A 0.5,0.5 -90 0,1 2,1.5 A 0.5,0.5 -90 0,1 1.5,2 A 0.5,0.5 -90 0,1 1,1.5 A 0.5,0.5 -90 0,1 1.5,1 Z

Basically the path_d goes around clockwise and the real circle goes around counter clockwise, but with the correct arcs. So the we end up creating scooped arcs for all the parts of the circle, which then go even more haywire later when converted for some still undiagnosed reason.

**Comment by tatarize** [comment]
Okay, fixed. Sort of. I think it could still go clockwise vs. counter-clockwise.

But, the rotated and flipped value, are such that the sweep should invert, but the criteria was wrong.

![image](https://user-images.githubusercontent.com/3302478/194707192-6a9916fa-fe82-4e34-9afd-67926cf99637.png)

**Comment by tatarize** [comment]
The bug was in svgelements so it's fixed for 0.7.8 too as is marked in this version. But, was a problem in 0.8.x since both use the same main code.

**Comment by tatarize** [comment]
Fixed in main.
